modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Options/Options.es.js

```diff
@@ -515,4 +515,33 @@ describe('Options', () => {
 
 		expect(valueInput.value).toBe('Bar');
 	});
+
+	it('removes an option when click on remove button', () => {
+		const {container} = render(
+			<OptionsWithProvider
+				defaultLanguageId={themeDisplay.getLanguageId()}
+				editingLanguageId="pt_BR"
+				name="options"
+				onChange={jest.fn()}
+				spritemap={spritemap}
+				value={optionsValue}
+			/>
+		);
+
+		act(() => {
+			jest.runAllTimers();
+		});
+
+		const removeOptionButton = document.querySelector('.ddm-option-entry .close');
+
+		fireEvent.click(removeOptionButton);
+
+		act(() => {
+			jest.runAllTimers();
+		});
+
+		const options = container.querySelectorAll('.ddm-field-options');
+
+		expect(options.length).toEqual(2);
```

Comment:
@alinedoleron, the `optionsValue` has 2 options by default and this test aims to remove one of them, right? If so, shouldn't the expected value of `options.length` be 1 instead of 2? Can you please clarify?

Reply:
Actually I'm looking for the '.ddm-field-options' element. As we always have one default option, and option Value has 2 values, in total we have 3 elements of '.ddm-field-options' initially.

Reply:
I see, @alinedoleron. Thanks for your clarification. In order to make easier the test comprehension, let's check the number of `.ddm-field-options` before click on the remove button, like the following:
```
let options = container.querySelectorAll('.ddm-field-options');

expect(options.length).toEqual(3);

const removeOptionButton = document.querySelector('.ddm-option-entry .close');

fireEvent.click(removeOptionButton);

act(() => {
	jest.runAllTimers();
});

options = container.querySelectorAll('.ddm-field-options');

expect(options.length).toEqual(2);
```

Can you please do this change?

Thanks, Alininha. 🙂